client/src/components/EvaluationResults.vue

```diff
@@ -27,7 +27,7 @@ export default {
         'green lighten-1',
         'green darken-2',
         'green darken-4',
-      ];
+        'black '];
```

Comment:
This seems like a spurious change.

Reply:
Oh yes, I added that because I noticed on this set that the `EvaluationResults` component was rendering wrong for soem of our example data (which represented an edge case for the Evaluation Results). This change does actually affect the appearance of that component in certain cases.